.circleci/config.yml

```diff
@@ -63,19 +63,19 @@ jobs:
 
 workflows:
   version: 2
-  deploy-prod:
-    triggers:
-      - schedule:
-          cron: "0 15 * * 1-5"
-          filters:
-            branches:
-              only:
-                - master
-    jobs:
-      - find-build
-      - deploy:
-          requires:
-            - find-build
+#  deploy-prod:
```

Comment:
TOL: I wonder if yaml supports including other yaml files, in which case maybe we can refactor this to make it an easy 1-line change in the future.